### DISCUSSION TITLE
Role linked to priority/que skip

### DISCUSSION THREAD

**Discussion by Dazzyreil** [Kilvoctu/aiyabot#115] [Ideas]
Hi,

Love the bot and the work you're putting into it, thanks for that!
As for my idea, is it possible to have a priority que linked to discord role? Lets say someone with the role "gold member" requests a /draw, right now his request gets put in the que at the end. I would be nice if roles could get inserted somewhere in the middle or top of the que.

**Comment by Kilvoctu** [comment]
Heya, glad you're having fun with the aiyabot 👍👍  
Yes it should possible to give priority to certain roles, but it may take quite a while for me to implement something like this. The queue code is still really spagetthi and would benefit from a few more refactors. I'll look into it when I have more time 👀
